charts/telepresence/templates/deployment.yaml

```diff
@@ -39,14 +39,25 @@ spec:
             value: app.getambassador.io
           - name: SYSTEMA_PORT
             value: "443"
+          - name: CLUSTER_ID
```

Comment:
so in this recent PR, I removed the CLUSTER_ID as an env variable and from the helm chart: https://github.com/telepresenceio/telepresence/pull/1986, so you should be able to remove everything in the helm chart that's related to CLUSTER_ID since that info is now gotten from the traffic-manager itself instead of passed in by the user or the cli on `connect`